app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def last_5_comments
+    comments.last(5)
+  end
+
+  def update_comments_count
+    update(comments_counter: comments.count)
```

Comment:
- [ ] Please include a method to increment the post counter. You can also add an active callback to increment after every new post is saved